.circleci/config.yml

```diff
@@ -7,6 +7,17 @@ jobs:
     steps:
       - checkout
 
+      - run:
+          name: Run audit-ci
+          command: |
+            export NVM_DIR="/opt/circleci/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install v8.11.4
```

Comment:
What version were we using before? And why do we need this one ?

Reply:
For the `machine` executor, the existing`nvm --version` is 0.31.0, `node --version` is v6.1.0, `npm --version` is 5.4.2, and we need to update the node to 8 and npm to 6 for the audit.